paypal/paypal-checkout-components
# chore(axo): Remove AXO

### Description
Removes Accelerated XO

### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues
Deprecating Accelerated XO V1
https://paypal.atlassian.net/browse/DTXOUP-984

### Testing
1. Verify correct rendering of button stack
2. Verify experience prop has no affect on rendering of button stack
3. Verify not errors in member, guest and inline guest flows.

❤️ Thank you!
